frontend/app/components/transportation/study-area-map.js

```diff
@@ -24,19 +24,9 @@ export default class TransportationProjectMapComponent extends Component {
     }
   }
 
-    /**
-   * The composite array of all highlighted features, including:
-   * - currently hovered feature
-   * - user-selected study selection features
-   * - required study selection features
-   * which is passed to the highlight layer's FeatureFilterer
-   */
-  @computed('hoveredFeatureId', 'analysis.{jtwStudySelection.[],requiredJtwStudySelection.[]}')
-  get highlightedFeatureIds() {
-    const { hoveredFeatureId } = this;
+  @computed('analysis.jtwStudySelection.[]')
+  get jtwStudySelectionComputed() {
```

Comment:
This computed was necessary to trigger `didRecieveAttrs` hook in the ```FeatureFilterer``` component. If directly bind the array to the filter component as `featureIds`, adding/removing things will not trigger the lifecycle hook.